components/Base/FileUpload.vue

```diff
@@ -52,8 +52,12 @@ const handleDragLeave = (event: DragEvent) => event.preventDefault();
 const handleDragOver = (event: DragEvent) => event.preventDefault();
 const handleDrop = (event: DragEvent) => {
   if (event.dataTransfer) {
-    selectedFile.value = event.dataTransfer.files[0];
-    emit('update:file', selectedFile.value);
+    const acceptedList = props.accept.split(',');
+    const loadFile = event.dataTransfer.files[0];
+    if (acceptedList.some((i) => loadFile.name.includes(i))) {
+      selectedFile.value = loadFile;
+      emit('update:file', selectedFile.value);
+    }
```

Comment:
I think here would be great to show some message that this file is not allowd, WDYT?